### PR TITLE
Create meeting ical for Azure working group

### PR DESCRIPTION
**Pull request by kyliel**

### CODE REVIEW

**Comment by kyliel** [comment]
@gundalow Could you please have a review? Thank you.

**Comment by kyliel** [comment]
@gundalow @nitzmahone we kicked off this meeting since last week. Could you please review the PR for ical? Then we could let more contributors know this meeting. Thank you.

**Comment by kyliel** [comment]
@gundalow and @nitzmahone , could you please help review the PR which is to set up ical for Azure working group as well as corresponding changes? Need your support to move forward. Thank you.

File: meetings/azure.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+chair: Matt Davis (nitzmahone), Kylie Liang (kyliel)
+description: |
+
+  We discuss everything related to Azure every week on Wednesday.
+
+  If you have something to discuss (e.g. a PR that needs help), add your
+  request to the meeting agenda and join the IRC channel.
+schedule:
+- time: '0000'
+  day: Wednesday
```
Comment (jborean93) [inline meetings/azure.yaml]: Isn't it technically Thursday at midnight (the morning of Thursday). These meetings are at 10am on a Thursday for me and I'm UTC+10. The ics would have to be changed.

**Comment by gundalow** [comment]
@kyliel Looks OK apart from the timezone

**Comment by kyliel** [comment]
ready_for_review

**Comment by kyliel** [comment]
@gundalow ready_for_review

**Pull request merged**